src/components/home/MapsContainer.vue

```diff
@@ -86,101 +73,100 @@
 
 <script>
 import { mapActions, mapGetters } from 'vuex';
-import HomeCard from '@/components/home/card/HomeCard';
+import HomeCard from '../../components/home/card/HomeCard';
```

Comment:
I believe the @/ alias should be kept, as I think it looks cleaner, and Vite supports aliases natively. (See https://vitejs.dev/config/shared-options.html#resolve-alias)

Reply:
Done